src/Modules/OrchardCore.Commerce/Assets/Scripts/stripe-card-form.js

```diff
@@ -69,7 +69,7 @@ function stripeCardForm(stripe, urlPrefix, fetchErrorText) {
                 .then(handleStripeJsResult);
         } else {
             // Show success message.
-            window.location.href = `${urlPrefix}/success`;
+            window.location.href = `${urlPrefix}/success/${response.OrderContentItemId}`;
```

Comment:
It sends me to https://localhost:5001/success/undefined. It's because this should be `response.orderContentItemId`? Like I said in the other thread, `return Json(new { ... })` automatically converts the result into camelCase.